docs/en/operations/clickhouse-keeper.md

```diff
@@ -105,196 +109,6 @@ ClickHouse Keeper is bundled into the ClickHouse server package, just add config
 clickhouse-keeper --config /etc/your_path_to_config/config.xml --daemon
 ```
 
-## Four Letter Word Commands
```

Comment:
Why this part removed?

Reply:
Sorry, no idea. I never touched anything apart from my task. And those lines weren't part of it.